### PR TITLE
Item crud

### PR DESCRIPTION
**Pull request by sandisz-d734m37**
__X__ Wrote Tests
__X__ Implemented
__AW__ First Review

## Necessary checkmarks:
- [X] All Tests are Passing
- [X] The code will run locally

## Type of change
- [X] New feature
- [] Bug Fix

## Implements/Fixes:
* Adds all Create, Update, and Destroy functionality for items
* __*IMPORTANT NOTE: Delete test actually deleted an item being used for many many tests. If VCRs are re-recorded, these tests will fail. A fix for this will come in my next PR.*__

## Check the correct boxes
- [] This broke nothing
- [X] All Tests are Passing
- [] The code will run locally

## Testing Changes
- [X] Tests have been added
- [X] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:
- [X] My code has no unused/commented out code
- [X] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [X] I have fully tested my code
- [X] I have removed any testing syntax (save_and_open_page/pry) in my spec files

### CODE REVIEW

File: app/poros/item.rb [hunk 1]
```diff
@@ -6,7 +6,24 @@ def initialize(data)
     @name = data[:attributes][:name]
     @description = data[:attributes][:description]
     @count = data[:attributes][:count]
-    @category = data[:attributes][:category]
+    @category = enum_category(data[:attributes][:category])
```
Comment (AliciaWatt) [inline app/poros/item.rb:9]: it might be weird but it works!

File: app/controllers/dashboard_controller.rb [hunk 2]
```diff
@@ -1,5 +1,7 @@
 class DashboardController < ApplicationController
   def show
     @user = UserFacade.user(session[:user_id])
+    @items = ItemFacade.items(session[:user_id])[0..2]
+    @trips = GearUpFacade.user_trips(session[:user_id]).sort_by{|trip| trip.start_date}[0..1]
```
Comment (AliciaWatt) [inline app/controllers/dashboard_controller.rb:5]: do we want this to stay in the GearUpFacade or move to TripFacade?

**Comment by AliciaWatt** [comment]
All tests pass. Item is crudable. Ready for merge after 2nd review.

**Comment by johnny-bowman** [comment]
Looks great!

**Pull request merged**